packages/pn-personafisica-webapp/src/navigation/AARGuard.tsx

```diff
@@ -52,6 +52,14 @@ const AARGuard = () => {
     }
   }, [aar]);
 
+  useEffect(() => {
+    AppResponsePublisher.error.subscribe('exchangeNotificationQrCode', ()=>false);
```

Comment:
the code is ok. we have to manage only the case of 404 and PN_DELIVERY_NOTIFICATIONNOTFOUND. In the function ()=>false, you should have the error object and you can use it to check if the error is the error with code PN_DELIVERY_NOTIFICATIONNOTFOUND

Reply:
doine